### PR TITLE
DOC-659 | Windows and macOS

### PR DESCRIPTION
**Pull request by nerpaula**
### Description

- https://github.com/arangodb/arangodb/pull/20636/
- https://github.com/arangodb/enterprise/pull/1452
- https://github.com/arangodb/arangodb/pull/20644

TODOs:
- [x] Clarify production-readiness of ArangoDB on Windows using containers
- [x] Simplify statements about x86-64 being for production and ARM for evaluation and testing
- [x] Remove or merge admonitions on the installation page

#### Upstream PRs

- 3.10:
- 3.11:
- 3.12:

### CODE REVIEW

**Comment by Simran-B** [comment]
Windows-specific startup options that get removed from arangod:
- `--start-service`
- `--install-service`
- `--uninstall-service`
- `--servicectl-start`
- `--servicectl-start-wait`
- `--servicectl-stop`
- `--servicectl-stop-wait`

And for arangosh:
- `--console.code-page`

They won't show up in the scheduled workflow because they are not available at all in Linux builds. This also means they never appeared in the auto-generated startup option list. There was also no other mention in the docs it seems.

The `--random.generator` startup option of arangod had an already deprecated option for WinCrypt (`5`) which gets removed.

**Comment by Simran-B** [comment]
https://deploy-preview-473--docs-hugo.netlify.app/devel/get-started/on-premises-installation/ should be changed to not mention Windows / show examples with Windows paths and probably changed to explain the available options (bare metal Linux and Docker), and that you can use Docker Desktop to run ArangoDB under Windows and macOS.

---

`--dump-options ` and the new startup option endpoints report operating systems under `os` which defaults to `["linux", "macos", "windows"]`. Jan will make a change in his PR to not report Windows anymore. macOS to be removed later as well.

The `program-options.md` shortcode needs to be adjusted because there is a check whether the reported OS list has 3 elements, which will become 2 with the Windows removal PR. As macOS will also be removed, there is no point in building up and displaying the OS info at all but we need to keep the functionality for 3.10 and 3.11.

--> Pushed a commit for this (no more mentions of "Linux only" or "Linux and macOS only" in the startup option overview for 3.12)

File: site/content/3.12/operations/installation/_index.md
Comment (Simran-B) [inline site/content/3.12/operations/installation/_index.md:1]: Note to self: there are a lot of admonitions now - consider merging them or pulling the content out

**Comment by Simran-B** [comment]
The random options are apparently in the server and client tools

**Comment by Simran-B** [comment]
Removed error code:

```yaml
- name: ERROR_OUT_OF_MEMORY_MMAP
  text: "out of memory in mmap"
  desc: "Will be raised when there is a memory shortage."
  code: 12
```

File: site/content/3.12/get-started/on-premises-installation.md [hunk 1]
```diff
@@ -9,17 +9,12 @@ description: >-
 ## Installation
 
 Head to [arangodb.com/download](https://www.arangodb.com/download/),
-select your operating system and download ArangoDB. You may also follow
+select your operating system, and download ArangoDB. You may also follow
```
Comment (ansoboleva) [inline site/content/3.12/get-started/on-premises-installation.md:12]: love for the oxford comma!

File: site/content/3.12/operations/installation/_index.md [hunk 2]
```diff
@@ -32,21 +27,20 @@ please refer to the [Deploy](../../deploy/_index.md) chapter.
...
 
-{{< tip >}}
-[ArangoGraph Insights Platform](https://dashboard.arangodb.cloud/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic)
-is a fully-managed service and requires no installation. It's the easiest way
-to run ArangoDB in the cloud.
-{{< /tip >}}
+Starting with version 3.12, ArangoDB packages for Windows and macOS are not provided
+anymore. You can use the official [Docker images](https://hub.docker.com/_/arangodb/)
+instead. Running production environments with Windows or macOS as the host
+operating system is not supported.
```
Comment (ansoboleva) [inline site/content/3.12/operations/installation/_index.md:37]: Please format it as a warning to make more prominent.

**Review by ansoboleva — APPROVED** [review]
please see the comment about adding a warning, otherwise lgtm

**Pull request merged**